src/lib/homestore_backend/hs_homeobject.cpp

```diff
@@ -135,14 +137,9 @@ void HSHomeObject::on_shard_meta_blk_found(homestore::meta_blk* mblk, sisl::byte
         add_new_shard_to_map(std::make_unique< HS_Shard >(sb));
         return;
     }
-
```

Comment:
Line 128 ~ 137 is no need anymore, but just keep L135

Reply:
ack